### DISCUSSION TITLE
NaN in weather card

### DISCUSSION THREAD

**Discussion by gdlesage** [cytech/Home-Assistant-wundergroundpws#128] [Q&A] [answered]
what data is shown in spot marked  in blue, my weather card is showing NaN (not a number).  This spot usually shows high and low temperature for current day, but why is mine showing NaN?
![Screenshot (29)](https://user-images.githubusercontent.com/14796404/206581591-77cad971-b164-4af9-96a8-b1afeef4cee4.png)

**Comment by gdlesage** [comment, marked as answer]
fixed itself with some time, now shows high/low temp

**Comment by cytech** [comment]
closed as outdated

**Comment by cytech** [comment]
closed as outdated
